### PR TITLE
Apply clippy suggestions for 1.67

### PR DESCRIPTION
**Pull request by djc**
Set an explicit `rust-version` to avoid further clippy-induced MSRV bumps.

### CODE REVIEW

File: testing/tests/filters.rs [hunk 1]
```diff
@@ -122,7 +122,7 @@ mod filters {
     }
     // for test_nested_filter_ref
     pub fn mytrim(s: &dyn (::std::fmt::Display)) -> ::askama::Result<String> {
-        let s = format!("{}", s);
+        let s = format!("{s}");
         Ok(s.trim().to_owned())
```
Comment (vallentin) [inline testing/tests/filters.rs:126]: Alternatively:

```suggestion
        Ok(s.to_string().trim().to_owned())
```

**Review by vallentin — APPROVED** [review]
One tiny nit, but otherwise looks fine :)

**Review by Kijewski — APPROVED** [review]

File: testing/tests/filters.rs (hunk 1 above)
Comment (Kijewski) [reply testing/tests/filters.rs:126]: Since it's just a test, I have no opinion either way. :) The one line is a bit more idiomatic.

Comment (djc) [reply testing/tests/filters.rs]: Nice, squashed that into my change!

**Review by Kijewski — APPROVED** [review]

**Pull request merged**